test/cbmc/proofs/ARP/ARPGenerateRequestPacket/ARPGenerateRequestPacket_harness.c

```diff
@@ -24,9 +28,15 @@ void harness()
     NetworkBufferDescriptor_t xNetworkBuffer2;
 
     xNetworkBuffer2.pucEthernetBuffer = xBuffer;
-    xNetworkBuffer2.xDataLength = ucBUFFER_SIZE;
 
-    /* vARPGenerateRequestPacket asserts buffer has room for a packet */
+    xNetworkBuffer2.xDataLength = ucBUFFER_SIZE;
+    /* vARPGenerateRequestPacket asserts buffer has room for a packet. */
     __CPROVER_assume( xNetworkBuffer2.xDataLength >= sizeof( ARPPacket_t ) );
```

Comment:
Would it be better to give this an a constraint on ucBuffer_Size above?

Reply:
I agree. Thanks!